src/utilities/to-array.ts

```diff
@@ -3,9 +3,11 @@
  * @param values
  * @returns {*}
  */
-module.exports = function toArray(values) {
+export function toArray(
+  values: Record<string, number> | number[]
```

Comment:
Again, type should be `number | number[]`

Reply:
Well, I think this is all good here. `Record<string, number> | number[]` is fine.